src/parsers/pm_io/psse.jl

```diff
@@ -1137,6 +1137,15 @@ function _psse2pm_transformer!(pm_data::Dict, pti_data::Dict, import_all::Bool)
                 Zx_s = 1 / 2 * (br_x23_sysbase - br_x31_sysbase + br_x12_sysbase)
                 Zx_t = 1 / 2 * (br_x31_sysbase - br_x12_sysbase + br_x23_sysbase)
 
+                # See PSSE Manual 1.15.1 on T3W Notes
```

Comment:
Add more detail to make it easy to find: 
See PSSE Manual (Section 1.15.1 "Three-Winding Transformer Notes" of Data Formats file)